packages/esm-service-queues-app/src/queue-table/queue-table-metrics.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useQueueEntries, useQueueEntriesMetrics } from '../hooks/useQueueEntries';
+import QueueTableMetricsCard from './queue-table-metrics-card.component';
+import styles from './queue-table-metrics.scss';
+import { type Queue } from '../types';
+
+interface QueueTableMetricsProps {
+  selectedQueue: Queue;
+}
+
+function QueueTableMetrics({ selectedQueue }: QueueTableMetricsProps) {
+  const { t } = useTranslation();
+
+  const allowedStatuses = selectedQueue.allowedStatuses;
+  const { totalCount } = useQueueEntries({ queue: selectedQueue.uuid, isEnded: false });
```

Comment:
Should this also use the metric endpoint for consistency?

Reply:
Sure, np. I have updated the PR.